src/views/InProgress.vue

```diff
@@ -801,9 +805,18 @@ export default defineComponent({
       }
       this.addingBoxForOrderIds.splice(this.addingBoxForOrderIds.indexOf(order.orderId), 1)
     },
-    getShipmentPackageNameAndType(shipmentPackage: any, order: any) {
-      // TODO
-      return order.shipmentBoxTypeByCarrierParty[shipmentPackage.carrierPartyId] ? `Box ${shipmentPackage.packageName} | ${this.boxTypeDesc(order.shipmentBoxTypeByCarrierParty[shipmentPackage.carrierPartyId][0])}` : ''
+    getShipmentPackageType(shipmentPackage: any, order: any) {
+      let packageType = '';
```

Comment:
We can set the default value for the packageType initially and can remove the extra nested if condition.
```suggestion
      let packageType = order.shipmentBoxTypeByCarrierParty[shipmentPackage.carrierPartyId][0];
```